muzlag/internal/service/player/player.go

```diff
@@ -0,0 +1,30 @@
+package player
+
+import (
+	"log/slog"
+
+	"github.com/Drozd0f/gobots/muzlag/pkg/ffmpeg"
+	"github.com/Drozd0f/gobots/muzlag/pkg/ytdl"
+)
+
+type NewServicePlayerParams struct {
+	Logger *slog.Logger
+
+	DL     ytdl.DL
+	Ffmpeg ffmpeg.Ffmpeg
+}
```

Comment:
Why do you need so much params structures?)

IMHO, for constructors better to use inline arguments, so you CANNOT miss something

Reply:
As for me, it's the other way around. Constructor can accept many arguments, and params can fix situation when

```go
func NewService(s1 someStruct1, s2 someStruct2, s3 someStruct3,
s4 someStruct4, s5 someStruct5, s6 someStruct6, s7 someStruct7,
s8 someStruct8, s9 someStruct9, s10 someStruct10, s11 someStruct11,
)
```

But I aggree with you about `miss something`

Reply:
But does your constructor will accept so many parameters?

Reply:
at the moment there is none and I will rewrite it into arguments